.changeset/techdocs-moose-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+Adjust the Tech Docs page theme as a side effect of the `mkdocs-material` theme update.
```

Comment:
I wonder if we should add a requirement here that one should be on a specific version of mkdocs-material or mkdocs-techdocs-core?

Reply:
Good idea, will do!

Reply:
Updated, looks good to you?

Reply:
Looks great! I wonder if its enough to focus on the docker image and say something like:

> If you use the `spofify/techdocs` image to build your documentation, make sure you use version `spotify/techdocs:v0.3.7`.....

as not everyone uses the cli to generate + publish docs! (and the cli will use the new version per default)